projects/kit/components/progress/progress.options.ts

```diff
@@ -0,0 +1,21 @@
+import type {Provider} from '@angular/core';
+import {tuiCreateToken, tuiProvideOptions} from '@taiga-ui/cdk';
+import type {TuiSizeL, TuiSizeS} from '@taiga-ui/core';
+
+export interface TuiProgressOptions {
+    readonly color: string | null;
+    readonly size: TuiSizeL | TuiSizeS;
```

Comment:
it is valid size?

Reply:
it's from here
https://github.com/taiga-family/taiga-ui/pull/8061/files#diff-e0d70b4f3901581cc0e5be478c2ea9e56e4368e40185ea6a5f0ef3fa400665e5

Reply:
fixed